WalletWasabi/TorDaemons/digests.txt

```diff
@@ -1,3 +1,3 @@
-5d5f175bf154f5f4cd2c460b9d4ea80c8219d07441f626e13a4d01c1408a0c28
-957768c0a005a9b6a1db1c52fc747ce39303415b412128731965ab3932c1fff4
-f6c8a3bd07b939e7c736a1e811df907611b763d0f8dbfafa721d8bf6ebfec691
\ No newline at end of file
+c53e24524b639635cddd4faea94ca2746ceb50352c4f1d2580e9e3115ec36c79
+48aeed60dfa0129fb34921b3b19d906bab395feded7c76de87b8e5cf4533e45e
+55840749c2c42b8079c2890b9ffdcff91cb93c40b378b43320fb9e4a72321842
```

Comment:
@yahiheb I can't recall how we get these digests. Can you share how you obtain them please?

Reply:
By running the following command:

WalletWasabi\WalletWasabi.Packager> `dotnet run -- onlycreatedigests`

https://github.com/zkSNACKs/WalletWasabi/blob/backport/WalletWasabi.Documentation/Guides/UpgradeTorDaemons.md#digest-creation